### PR TITLE
remove v2

### PR DESCRIPTION
**Pull request by terrazoon**
## Description

Remove v2 API since it doesn't seem to be used for anything.  Looking at Notify UK, there has been very little development in v2 since we forked, and the development that has occurred is in areas we are not interested in (email, physical mail).

Looking at all the calls to v2 apis in our logs, they seem to be unauthorized calls from scanners.  The only time we do call the v2 api, it is to retrieve a template, and we can just substitute the normal template call in that one spot.

## Security Considerations

Remove dead code to shrink attack surface.

### CODE REVIEW

File: app/errors.py [hunk 1]
```diff
@@ -113,3 +114,45 @@ def internal_server_error(e):
         e = getattr(e, "original_exception", e)
         current_app.logger.exception(e)
         return jsonify(result="error", message="Internal server error"), 500
+
+
+class TooManyRequestsError(InvalidRequest):
```
Comment (jskinne3) [inline app/errors.py:119]: This class does not appear to be used

**Review by jskinne3 — APPROVED** [review]
Looks good, just one nitpick

File: app/errors.py (hunk 1 above)
Comment (ccostino) [reply app/errors.py:119]: Thanks for flagging, @jskinne3!  While that might be the case, I could see it being used in the not-too-distant future when we revisit message limits.  We can leave in for now and remove it later if it turns out we truly don't need it.

File: tests/app/user/test_rest_verify.py [hunk 2]
```diff
@@ -500,10 +500,10 @@ def test_send_user_email_code_with_urlencoded_next_param(
...
     mock_redis_get.return_value = "foo"
 
-    mock_s3_personalisation = mocker.patch(
-        "app.v2.notifications.get_notifications.get_personalisation_from_s3"
-    )
-    mock_s3_personalisation.return_value = {"name": "Bob"}
+    # mock_s3_personalisation = mocker.patch(
+    #    "app.v2.notifications.get_notifications.get_personalisation_from_s3"
+    # )
+    # mock_s3_personalisation.return_value = {"name": "Bob"}
```
Comment (ccostino) [inline tests/app/user/test_rest_verify.py:506]: It seems like we can just get rid of these lines, right?

```suggestion
```

**Review by ccostino — COMMENTED** [review]
Thank you, @terrazoon, for both looking into how we may have diverged from UK Notify and cleaning this up on our side!

There was only one additional thing that caught my eye in terms of commented out lines left in place, otherwise I think we're good to go. 🙂

File: tests/app/user/test_rest_verify.py (hunk 2 above)
Comment (ccostino) [reply tests/app/user/test_rest_verify.py:506]: @terrazoon are there any concerns if we just remove these commented out lines?

**Review by ccostino — APPROVED** [review]
Thanks, @terrazoon!

**Pull request merged**